### ISSUE TITLE
No qualifying bean of type [org.springframework.transaction.PlatformTransactionManager]

### ISSUE THREAD

**Issue by YaraslauBarysenka** [javers/javers#476]
### Preconditions

- javers version is 2.7.2;
- using javers spring integration (please see JaversConfiguration.java file in the application example);
- two transaction managers are available in spring context;
- using org.javers.spring.jpa.TransactionalJaversBuilder to build a org.javers.core.Javers object.

### Current Result
We got excaption like this: Caused by: org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type [org.springframework.transaction.PlatformTransactionManager] is defined: expected single matching bean but found 2: transactionManager,secondTransactionManager

### Expected Result
Javers library provides avalible method to set PlatformTransactionManager when we build a org.javers.core.Javers object. 

If Javers can do this could you please provide information how to do that?

## Runnable test case
The application to reproduce issue here: [javers-bug-example.zip](https://github.com/javers/javers/files/636992/javers-bug-example.zip)

In order to reproduce the issue please execute "mvn clean package" in root folder.

### P.S.
If you have questions, please let's me know.
Thank you in advance.

**Comment by bartoszwalacik** [comment]
Thanks for hight quality report, we will add support for selecting transaction manager

**Comment by bartoszwalacik** [comment]
released in 2.8.0, now you pass concrete txManager instance (no autowiring) :

```java
    @Bean
    public Javers javers(PlatformTransactionManager txManager) {
        JaversSqlRepository sqlRepository = SqlRepositoryBuilder
                .sqlRepository()
                .withConnectionProvider(jpaConnectionProvider())
                .withDialect(DialectName.H2)
                .build();

        return TransactionalJaversBuilder
                .javers()
                .withTxManager(txManager)
                .withObjectAccessHook(new HibernateUnproxyObjectAccessHook())
                .registerJaversRepository(sqlRepository)
                .build();
    }
```
